microcloud/test/suites/preseed.sh

```diff
@@ -29,6 +29,7 @@ ovn:
   ipv4_gateway: 10.1.123.1/24
   ipv4_range: 10.1.123.100-10.1.123.254
   ipv6_gateway: fd42:1:1234:1234::1/64
+  dns_servers: 10.1.123.1,8.8.8.8
```

Comment:
please help me understand where the tests check that these settings are applied to LXD?

Reply:
For now, there's no check. I'll add that, thanks!

Reply:
should be added now